src/components/Gallery.js

```diff
@@ -25,39 +31,45 @@ function renderImage(item) {
             ))
           }
           <img
-            alt={item.originalAlt}
+            alt={originalAlt}
             src={item.original}
           />
         </picture>
       :
         <img
           src={item.original}
-          alt={item.originalAlt}
+          alt={originalAlt}
           srcSet={item.srcSet}
           sizes={item.sizes}
           title={item.originalTitle}
-          //onLoad={this.props.onImageLoad}
-          //onError={onImageError}
         />
     }
 
     {
       item.description &&
         <span className='image-gallery-description'>
-          {item.description}
+          <FormattedMessage id={item.description}/>
         </span>
     }
   </div>
 
   );
 }
 
-const Gallery = ( { images } ) => (
-  <ImageGallery lazyLoad={true} showBullets={true} renderItem={renderImage} items={images} />
-)
+const Gallery = ( { images, intl } ) => {
+  const placeholder = intl.formatMessage({id: 'imageAlt01'});
```

Comment:
this is only for testing if intl si passed. Infact placeholder is correctly injected with the right values. The result:
for the italian message: 
"Oh! Delle formiche!"
en message:
"Oh! Some ants!"